electricitylci/olca_jsonld_writer.py

```diff
@@ -251,17 +251,17 @@ def clean_json(file_path):
             data['Flow']['ids'].pop(idx)
             data['Flow']['objs'].pop(idx)
 
-        # Update flow w/ FEDEFL metadata
-        # https://github.com/USEPA/ElectricityLCI/discussions/239
-        fed_version = _get_fedefl_version()
-        for i in range(len(data['Flow']['objs'])):
-            uid = data['Flow']['objs'][i].id
-            fed_cas, fed_form, fed_syn = _get_fedefl_meta(uid)
-            data['Flow']['objs'][i].cas = fed_cas
-            data['Flow']['objs'][i].formula = fed_form
-            data['Flow']['objs'][i].synonyms = fed_syn
-            data['Flow']['objs'][i].description = (
-                "Based on FEDELEMFLOW version %s" % fed_version)
+        # # Update flow w/ FEDEFL metadata
+        # # https://github.com/USEPA/ElectricityLCI/discussions/239
+        # fed_version = _get_fedefl_version()
+        # for i in range(len(data['Flow']['objs'])):
+        #     uid = data['Flow']['objs'][i].id
+        #     fed_cas, fed_form, fed_syn = _get_fedefl_meta(uid)
+        #     data['Flow']['objs'][i].cas = fed_cas
+        #     data['Flow']['objs'][i].formula = fed_form
+        #     data['Flow']['objs'][i].synonyms = fed_syn
+        #     data['Flow']['objs'][i].description = (
```

Comment:
@bl-young, where is the FEDEFL version recorded in your proposed solution?

Reply:
Within fedelemflowlist it gets added:
https://github.com/USEPA/fedelemflowlist/blob/7f2550a383a026ade3d23c8b0e3f6d1ba411bdd3/fedelemflowlist/jsonld.py#L154-L160